minigui/layer.ts

```diff
@@ -458,7 +458,20 @@ namespace Variation {
 
 
 class Annotations extends Layer {
-  private annotations = new Map<Annotation.Shape, Annotation[]>();
+  private annotations = new Map<string, Annotation[]>();
```

Comment:
as mentioned below, `annotations` can just be an `Annotation[]`, no need for a map

Reply:
Done.